tests/test_filings.py

```diff
@@ -108,6 +108,21 @@ def test_sort_bad_column(self):
         response = self.app.get(api.url_for(FilingsList, sort='request_type'))
         self.assertEqual(response.status_code, 422)
 
+    def test_sort_multi_column(self):
+        [
+            factories.FilingsFactory(beginning_image_number=2, coverage_end_date='2017-01-01'),
+            factories.FilingsFactory(beginning_image_number=1, coverage_end_date='2017-01-01'),
+            factories.FilingsFactory(beginning_image_number=0, coverage_end_date='2017-01-02'),
+        ]
+
+        results = self._results(api.url_for(FilingsList, sort=['coverage_end_date', 'beginning_image_number']))
+        self.assertEqual(results[0]['beginning_image_number'], '1')
+        self.assertEqual(results[2]['beginning_image_number'], '0')
```

Comment:
Ok, actually, I've got a couple of questions before I can contribute. (And this points to the clarity I think we can add with some edits.) 

What are these asserts testing for? Is it that when sorting by multiple fields, the secondary field is correctly ordered?